app/models/accounting/quickbooks/transaction_creator.rb

```diff
@@ -32,20 +34,24 @@ def add_disbursement(amount:, loan_id:, memo:, description:, qb_bank_account_id:
           posting_type: 'Credit',
           description: description,
           qb_account_id: qb_bank_account_id,
-          qb_customer_ref: qb_customer_ref
+          qb_customer_ref: qb_customer_ref,
+          qb_department_ref: qb_department_ref
         )
 
         service.create(je)
       end
 
       private
 
-      # Not memoized because organization could vary. Make sure to capture in an ivar,
```

Comment:
Changed to take a param. I don't think the comment make sense. You wouldn't usually memoize a method with a param.